gurubu-client/src/app/styles/page/style.scss

```diff
@@ -4,3 +4,10 @@
 // Sections
 @import "./hero-section.scss";
 @import "./testimonial.scss";
+@import "./pricing.scss";
+
+main {
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
why did you add this code block ?

Reply:
That was already there, I didn't add that. Could you check it out again please?

Reply:
I think linter issue ok then LGTM for me I will merge it!